src/collective/sidebar/browser/overrides/plone.app.layout.viewlets.sections.pt

```diff
@@ -0,0 +1,31 @@
+<tal:sections xmlns:tal="http://xml.zope.org/namespaces/tal" xmlns:metal="http://xml.zope.org/namespaces/metal" xmlns:i18n="http://xml.zope.org/namespaces/i18n" tal:define="portal_tabs view/portal_tabs" tal:condition="portal_tabs" i18n:domain="plone">
```

Comment:
The `tal:condition="portal_tabs"` should be removed so that the burger menu is still accessible even when no site links exist.